### ISSUE TITLE
Why not shuffling the train_loader_out?

### ISSUE THREAD

**Issue by zhmiao** [wetliu/energy_ood#2]
Hello! Thank you very much for your good work. I have a small question about the implementation. I realize that the train_loader_out is not shuffling. Instead, you use a random offset to induce the randomness. Could you please clarify the reason for this operation? Thanks again!

**Comment by weitliu** [comment]
Thank you for your question. Our code is based on the implementation of outlier-exposure which utilizes the auxiliary dataset this way. We did try to change it but it will change the results compared to OE, which sets the seeds. You can checkout the reason(s) in OE's issues if there are answers (link in README).
